src/include/rrdb/rrdb.client.h

```diff
@@ -417,6 +417,17 @@ class rrdb_client
         return rpc.call(_resolver, tracker, std::forward<TCallback &&>(callback));
     }
 
+    // ---------- call RPC_DETECT_HOTKEY ------------
+    // - synchronous
+    std::pair<::dsn::error_code, hotkey_detect_response>
+    detect_hotkey_sync(const hotkey_detect_request &args,
+                       std::chrono::milliseconds timeout,
+                       uint64_t partition_hash)
+    {
+        return ::dsn::rpc::wait_and_unwrap<hotkey_detect_response>(_resolver->call_op(
+            RPC_DETECT_HOTKEY, args, &_tracker, empty_rpc_handler, timeout, partition_hash));
```

Comment:
RPC_DETECT_HOTKEY should not be a rrdb task code, see comments in `rrdb.code.definition.h`